### PR TITLE
creating sessionID

### PR DESCRIPTION
**Pull request by larrycoal**
i used express-session to create sessionID  for each user and exported userSession funtion that returned the ID

### CODE REVIEW

File: src/helpers/express_session.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const app = express()
+const session = require('express-session')
+
+app.use(session({
+    secret:'urlShortner',
+    saveUninitialized:true,
+    resave:true
+}))
+var userId;
```
Comment (seyiogunjuyigbe) [inline src/helpers/express_session.js:10]: Hey bro, kindly point the line that extends the session length indefinitely

Comment (larrycoal) [reply src/helpers/express_session.js:10]: session length by default is indefinite,unless i change it

Comment (larrycoal) [reply src/helpers/express_session.js:10]: if u need me to specifically set it,all i need to do is set makAge,but by default it is nulll which means it dosent expire

**Review by seyiogunjuyigbe — APPROVED** [review]

**Comment by code2exe** [comment]
larrycoal, your repo has node_modules

**Comment by code2exe** [comment]
Please fix that and resend a PR

**Pull request merged**